gravitee-am-model/src/main/java/io/gravitee/am/model/User.java

```diff
@@ -149,7 +149,10 @@ public class User implements IUser {
     @Schema(type = "java.lang.Long")
     private Date updatedAt;
 
+    private Boolean forceResetPassword = Boolean.FALSE;
+
     public User() {
```

Comment:
can't it be Lombok used?

Reply:
there is too many custom getter and setters.